### DISCUSSION TITLE
2.6更新以后的几个问题

### DISCUSSION THREAD

**Discussion by oTnTh** [morlay/genshintools#15] [Q&A]
我在PC端导出新的抽卡记录，然后导入GT，程序显示成功，但是手机端的抽卡记录页面时一片空白，也没报告任何错误。

目前我这里的app版本是2.5.0 build 220314119，想着是不是要更新一下，然后gitee删掉了这里也没有。

也不确定这个是不是bug所以就跑这里来问你了。

**Comment by morlay** [comment]
在 release-stable 分支

**Comment by oTnTh** [comment]
搞定了，更新完了就好了。

Thanks～
